jira-assistant-base/src/main/java/net/netconomy/jiraassistant/base/services/issues/BasicIssueService.java

```diff
@@ -86,19 +89,22 @@ public Issue getIssue(ClientCredentials credentials, String issueKey, Boolean wi
             Boolean withChangelog) {
 
         Map<String, Issue> issueMap = new HashMap<>();
+        if (CollectionUtils.isEmpty(issueKeyList)) {
+            return issueMap;
+        }
 
         String changeLogString;
 
         try (JiraRestClient client = clientFactory.create(credentials);) {
 
             if (LOGGER.isInfoEnabled()) {
                 if (withChangelog) {
-                    changeLogString = "with Changelog";
+                    changeLogString = "with changelog";
                 } else {
-                    changeLogString = "without Changelog";
+                    changeLogString = "without changelog";
                 }
 
-                LOGGER.info("Retrieving {} Issues from Jira, {}.", issueKeyList.size(), changeLogString);
+                LOGGER.info("Retrieving {} issues from Jira {}.", issueKeyList.size(), changeLogString);
```

Comment:
Couldn't you do something like `LOGGER.INFO("Retrieving {} issues from Jira {} changelog", issueKeyList.size(), withChangelog ? "with" : "without")`